src/CryptoNoteCore/CachedBlock.cpp

```diff
@@ -102,6 +106,28 @@ const BinaryArray& CachedBlock::getBlockHashingBinaryArray() const {
   return blockHashingBinaryArray.get();
 }
 
+const BinaryArray& CachedBlock::getSignedBlockHashingBinaryArray() const {
```

Comment:
This appends signature to block hashing blob so pool operator can not sign the block, because signature changes POW hash